.sonarcloud.properties

```diff
@@ -0,0 +1,9 @@
+# Path to sources
+sonar.sources=.
+sonar.exclusions=**/*test*, **/vendor/**, **/zz_generated.*, **/*.yaml,
```

Comment:
this repo mostly contains pipelines and tasks definitions, there is no golang code here, will it be good to run sonar cloud check only on the `python` or `shell` scripts present in this repo?